notebooks/week3/03.model_serving_feature_lookup.py

```
@@ -0,0 +1,160 @@
+# COMMAND ----------
+
+
+
+train_set = spark.table(f"{catalog_name}.{schema_name}.train_set").toPandas()
+
+sampled_records = train_set[required_columns].sample(n=1000, replace=True).to_dict(orient="records")
+dataframe_records = [[record] for record in sampled_records]
+
+# COMMAND ----------
+
+train_set.dtypes
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)